adobe/spectrum-web-components
# fix: support --spectrum-global-dimension-dividers token

## Description
Add `--spectrum-global-dimension-dividers` statically to the various scale systems.

## Related issue(s)

- fixes #1107

## Motivation and context
Easier to make applications

## How has this been tested?

    1. Go to https://dividers--spectrum-web-components.netlify.app/storybook/?path=/story/styles--dividers
    2. See that the divider width is 1px when `scale=large` and 2px when `scale=medium`

## Types of changes

## Checklist

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
